mimic/rest/maas_api.py

```diff
@@ -118,7 +118,7 @@ def create_entity(params):
     """
     params = collections.defaultdict(lambda: '', params)
     newentity = {}
-    newentity['label'] = params[u'label'].encode("utf-8")
+    newentity['label'] = params['label']
```

Comment:
Updated docstrings with cross references to the API docs and explanation of key/value types.